.github/workflows/comment_iss-close.yml

```
@@ -0,0 +1,29 @@
+on:
+  issues:
+    types: [closed]
+
+jobs:
+  greet-on-close:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: Greet User
+        uses: actions/github-script@v5
```

actionlint

the runner of "actions/github-script@v5" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)